lib/parsers/bmp_parser.rb

```diff
@@ -0,0 +1,41 @@
+class FormatParser::BMPParser
+  include FormatParser::IOUtils
+
+  VALID_BMP = 'BM'
+  PIXEL_ARRAY_OFFSET = 54
+  COLOR_TYPES = {
+    1 => :monochrome
```

Comment:
Could the data available here be useful? http://www.dragonwins.com/domains/getteched/bmp/bmpfileformat.htm#The%20Color%20Table

Reply:
@julik after some read and some experimentation by using ImageMagick to convert `png` with alpha to `bmp` and checking the content of the `io` object and been the same I decide to make the color mode `:rgb` by default. It would be much faster since there is no need to read more from the `io` object. Please correct me if I'm wrong. Thanks